packages/kbn-optimizer/limits.yml

```diff
@@ -105,7 +105,7 @@ pageLoadAssetSize:
   navigation: 37269
   newsfeed: 42228
   noDataPage: 5000
-  observability: 118191
+  observability: 160191
```

Comment:
Did we look into reducing this?

Reply:
I'm not sure I can with the addition of add to case flyout from the cases plugin, at least without it being pulled out into a package. cc: @cnasikas @michaelolo24

Reply:
In syncing with Christos last week, the option of moving it out to a package has been proposed, so definitely something we can investigate